containerbuddy/onchange_reload-nfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
```

Comment:
You want to use `$CONSUL_HOST/v1/health/service/$SERVICE_NAME?passing` here? See also https://github.com/autopilotpattern/wordpress/pull/3#discussion_r58459297